sonic_platform_base/chassis_base.py

```diff
@@ -30,6 +32,8 @@ class ChassisBase(device_base.DeviceBase):
     REBOOT_CAUSE_HARDWARE_RESET_FROM_ASIC = "Reset from ASIC"
     REBOOT_CAUSE_NON_HARDWARE = "Non-Hardware"
 
+    SENSORS_YAML_FILE = "/usr/local/etc/sensors.yaml"
```

Comment:
Why not put the file under the platform folder? I suppose the sensor list will be platform-specific?

Reply:
The sensor list is platform specific, however the location of the file may not be. However I agree it might be better. I will look into this and get back.

Reply:
I have added the file under platform folder.